crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -1,4 +1,77 @@
-//! The workflow and queue consumer for sys validation
+//! ### The sys validation workflow
+//!
+//! This workflow runs against all [`Action`]s that are going into the DHT database. Either coming from the authored database or from other nodes on the network.
+//!
+//! The purpose of the workflow is to make fundamental checks on the integrity of the data being put into the DHT. This ensures that invalid data is not served
+//! to other nodes on the network. It also saves hApp developers from having to write these checks themselves since they set the minimum standards that all data
+//! should meet regardless of the requirements of a given hApp.
+//!
+//! #### Validation checks
+//!
+//! The workflow operates on [`DhtOp`]s which combine [`Action`]s give context to [`Action`]s and carry [`Entry`]s where relevant. Checks that you can rely on
```

Comment:
Alternate suggestion. Not ideal because I'm not sure if we really define "authority type" elsewhere in the docs but this is a bit more explanatory.

```suggestion
//! The workflow operates on [`DhtOp`]s which are roughly equivalent to [`Record`]s but catered to the needs of a specific type of Authority.
```

Reply:
Accepted by manual apply to avoid removing the rest of the line